tests/unit-core/test-arraybuffer.c

```diff
@@ -326,8 +326,6 @@ main (void)
       sum += data[i];
     }
 
-    jerry_release_value (buffer);
```

Comment:
Why was this removed?

Reply:
I feel this kind of implicit value acquire leads to unexpected memory leaks.

Reply:
Furthermore it cannot stop users to try to access the pointer after the buffer is freed.

Reply:
+1 for removing the extra reference. That's totally the user's responsibility to manage the buffer correctly.

Reply:
ahh true